examples/desktopvirtualization/virtualdesktopapplication.yaml

```
@@ -0,0 +1,86 @@
+apiVersion: azure.upbound.io/v1beta1
+kind: ResourceGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: virtualdesktop/v1beta1/desktophostpool
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    location: West Europe
+
+
+---
+
+apiVersion: virtualdesktop.azure.upbound.io/v1beta1
+kind: VirtualDesktopHostPool
```

Here and in all other examples there is a mismatch in group names and resource kinds.

```suggestion
kind: HostPool
```